packages/client/e2e/credit.spec.ts

```diff
@@ -58,7 +64,7 @@ test.describe("Credit", () => {
         .click()
 
       const quantity = newRfqPage.getByTestId("quantity")
-      await quantity.type("2")
+      await quantity.fill("2")
```

Comment:
a-ha - I heard about this on the Playwright update video .. need to be careful we use this in the right context, as it bypasses character input events - we do want to `type` in this case (not `fill`) as we have custom functionality which changes the formatting of the value as we type

Reply:
good point, I've updated all deprecated `type` to `pressSequentially` throughout the tests suite 👍